packages/blade/src/utils/storybook/Sandbox/baseCode.ts

```diff
@@ -0,0 +1,289 @@
+import packageJson from '../../../../package.json'; // eslint-disable-line
+
+const getBladeVersion = (): string => {
+  // We don't publish codesandbox ci on master so version is not present
+  const isMaster = process.env.GITHUB_REF === 'refs/heads/master';
+  const sha = process.env.GITHUB_SHA;
+  if (sha && !isMaster) {
+    const shortSha = sha.slice(0, 8);
+    return `https://pkg.csb.dev/razorpay/blade/commit/${shortSha}/@razorpay/blade`;
+  }
+
+  return '*';
+};
+
+export const tsConfigJSON = JSON.stringify(
+  {
+    compilerOptions: {
+      target: 'ES2020',
+      useDefineForClassFields: true,
+      lib: ['ES2020', 'DOM', 'DOM.Iterable'],
+      module: 'ESNext',
+      skipLibCheck: true,
+
+      /* Bundler mode */
+      moduleResolution: 'bundler',
+      allowImportingTsExtensions: true,
+      resolveJsonModule: true,
+      isolatedModules: true,
+      noEmit: true,
+      jsx: 'react-jsx',
+
+      /* Linting */
+      strict: true,
+      noUnusedLocals: true,
+      noUnusedParameters: true,
+      noFallthroughCasesInSwitch: true,
+    },
+    include: ['src'],
+  },
+  null,
+  4,
+);
+
+type Dependencies = {
+  dependencies: Record<string, string>;
+  devDependencies?: Record<string, string>;
+};
+
+export const getReactScriptsJSDependencies = (): Dependencies => {
+  return {
+    dependencies: {
+      react: '^18',
+      'react-dom': '^18',
+      'react-scripts': '4.0.3',
+      '@razorpay/blade': getBladeVersion(),
+      'styled-components': packageJson.peerDependencies['styled-components'],
+    },
+  };
+};
+
+const getViteReactTSDependencies = (): Dependencies => {
+  return {
+    dependencies: {
+      react: '^18',
+      'react-dom': '^18',
+      '@types/react': '^18',
+      '@types/react-dom': '^18',
+      '@razorpay/blade': getBladeVersion(),
+      'styled-components': packageJson.peerDependencies['styled-components'],
+    },
+    devDependencies: {
+      vite: '4.5.0',
+      '@vitejs/plugin-react': '4.1.1',
+    },
+  };
+};
+
+export const vitePackageJSON = JSON.stringify(
+  {
+    scripts: {
+      dev: 'vite',
+      build: 'vite build',
+    },
+    ...getViteReactTSDependencies(),
+  },
+  null,
+  4,
+);
+
+export const viteConfigTS = dedent`
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+})
+`;
+
+export const indexHTML = dedent`
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="https://raw.githubusercontent.com/razorpay/blade/1e77f0b35172654037a431a916b3190b545fd232/branding/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blade Example</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/index.tsx"></script>
+  </body>
+</html>
+`;
+
+export const logger = dedent`
```

Comment:
StackblitzEditor does not support browser console so this just injects a console and overrides the console.log inside editor to also log into this DOM console along with actual console